File: app/src/main/java/com/schrondinger/quin/base/mvp/BaseView.java
package com.schrondinger.quin.base.mvp;

/**
 * Created by hp on 2017/1/3.
 */

public interface BaseView {
    void onRequestStart(int tag);
    void onRequestError(int tag, String msg);
    void onRequestEnd(int tag);
    void onInternetError();
}

File: app/src/main/java/com/schrondinger/quin/base/mvp/BaseView.kt
package com.schrondinger.quin.base.mvp

/**
 * Created by hp on 2017/1/3.
 */

interface BaseView {
    fun onRequestStart(tag: Int)
    fun onRequestError(tag: Int, msg: String)
    fun onRequestEnd(tag: Int)
    fun onInternetError()
}
